<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PswGenDialog</class>
 <widget class="QDialog" name="PswGenDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор паролів</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Довжина паролю:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="pswLenSlider">
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pswLenSpin"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="useDigitsCheck">
     <property name="text">
      <string>Використовувати цифри</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useSpecsCheck">
     <property name="text">
      <string>Використовувати спеціальні символи</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ваш пароль:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pswEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Згенерувати!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
